frontend/src/pages/CourseSelector/CourseDescription/AddToPlannerButton.jsx

```diff
@@ -48,6 +48,7 @@ const AddToPlannerButton = () => {
         handbookNote: course.handbook_note,
         isAccurate: course.is_accurate,
         mark: null,
+        isMultiterm: course.is_multiterm,
```

Comment:
Replying here because I can't figure out how to reply directly. 

> Would it be better to drag all instance of that course individually rather than forcing the movement for all instance?

I considered doing this but multiterm courses need to be done consecutively (i.e. you can't do ENG2600 in 2020T1, 2020T3 and 2021T1). They are allowed to wraparound to the next year, though. I think it would mislead the user if they could break up a multiterm course, even if there was a warning saying they shouldn't. 

> also, marks, i think its using the same one? Any plans to encapsulate them for their own instance (or if there is a need?)

I don't think there's a need for this. Multiterm courses get given a mark once every instance has been completed (not 2 or 3 seperate marks). Students don't receive a mark for each term so they wouldn't expect to be able to enter a different mark for each term.